termuic/termuiConsoleGrid.go

```diff
@@ -0,0 +1,162 @@
+package termuic
+
+import (
+	"fmt"
+	"strconv"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/gizak/termui/v3/widgets"
+)
+
+type termuiConsoleGrid struct {
+	grid       *ui.Grid
+	lLog       *widgets.List
+	pPublicKey *widgets.Paragraph
+	pShardId   *widgets.Paragraph
+
+	pCountLeader         *widgets.Paragraph
+	pCountConsensus      *widgets.Paragraph
+	pCountAcceptedBlocks *widgets.Paragraph
+
+	gTxPoolLoad        *widgets.Gauge
+	pNumConnectedPeers *widgets.Paragraph
+
+	tSyncInfo *widgets.Table
+}
+
+//NewtermuiConsoleGrid initialize struct termuiConsoleGrid
+func NewtermuiConsoleGrid() *termuiConsoleGrid {
+
+	self := termuiConsoleGrid{}
+
+	self.initWidgets()
+
+	self.setupGrid()
+
+	return &self
+}
+
+func (tcg *termuiConsoleGrid) initWidgets() {
+
+	tcg.pPublicKey = widgets.NewParagraph()
+	tcg.pShardId = widgets.NewParagraph()
+
+	tcg.pCountLeader = widgets.NewParagraph()
+	tcg.pCountConsensus = widgets.NewParagraph()
+	tcg.pCountAcceptedBlocks = widgets.NewParagraph()
+
+	tcg.gTxPoolLoad = widgets.NewGauge()
+	tcg.pNumConnectedPeers = widgets.NewParagraph()
+
+	tcg.lLog = widgets.NewList()
+
+	tcg.tSyncInfo = widgets.NewTable()
+}
+
+func (tcg *termuiConsoleGrid) setupGrid() {
+	tcg.grid = ui.NewGrid()
+
+	tcg.grid.Set(
+		ui.NewRow(1.0/7, tcg.pPublicKey),
```

Comment:
a lot of magic numbers

Reply:
there a lot of magic numbers because in this section of code needs to specify the number of column for every row  or de number of row for every column, so for the moment for first version there a some magic number, if a change the view of console again the magic number need to be change.

Reply:
created a const for magic number 1.0 (1.0 means a full column or a full row in a grid)